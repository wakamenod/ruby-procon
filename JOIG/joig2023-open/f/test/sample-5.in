19
JOIGCODINGCHALLENGE
5
1 1 1
100 200 200
225 111 111
123456789 987654321 987654321
31415926535 27182818284 27182818284
