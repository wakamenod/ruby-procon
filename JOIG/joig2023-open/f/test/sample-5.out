48
7700
7494
30987654300
1385204334401
